src/GNDStk/BodyText/test/types.test.cpp

```diff
@@ -0,0 +1,88 @@
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+using namespace njoy::GNDStk::core;
+
+// -----------------------------------------------------------------------------
+// Scenario
+// -----------------------------------------------------------------------------
+
+SCENARIO("BodyText data types") {
+   GIVEN("std::vector<T> for certain specific Ts") {
+
+      // vector type
+      using vv = BodyText<true>::VariantOfVectors;
+      CHECK(std::variant_size_v<vv> == 15);
+      vv vectors;
+
+      // scalar type
+
+      using vs = BodyText<true>::VariantOfScalars;
+      CHECK(std::variant_size_v<vv> == 15);
+      vs scalars;
+
+      {
+         vectors = std::vector<std::string>(10,"a");
+         scalars = std::get<std::vector<std::string>>(vectors)[0];
+         CHECK(std::get<std::string>(scalars) == "a");
+      } {
+         vectors = std::vector<char>(10,'b');
+         scalars = std::get<std::vector<char>>(vectors)[0];
```

Comment:
same as above

Reply:
I've made these changes, in the files you mentioned here, and a handful of others in the code base. Changes are in my newest branch, which I'll upload shortly.